src/tests/unit/tests/DetailsView/components/failure-instance-panel-control.test.tsx

```diff
@@ -87,26 +91,28 @@ describe('FailureInstancePanelControlTest', () => {
     });
 
     test('onValidateSelector ', () => {
-        const selector = 'some selector';
-        const snippet = 'snippet for ' + selector;
+        const path = 'some selector';
```

Comment:
Might be able to move this definition into line 100 similar to how `failureDescription` is defined inline

Reply:
Got it! Good point--not using that variable anywhere else